kubeflow/example/readme.md

```diff
@@ -7,7 +7,17 @@ The main objective of this pipeline is to perform the model training for a drivi
 
 The pipeline is composed of the following components:
 
-### Downloading data
+###  Interacting with MinIO
```

Comment:
I personally would use just on title/section instead of two levels (### + ####). I would add just "### Connecting to MinIO"

Reply:
Done! LGFM